ffcx/codegeneration/integrals_generator.py

```diff
@@ -149,12 +148,49 @@ def generate(self):
                           "coordinate_dofs = (const double*)__builtin_assume_aligned(coordinate_dofs, {});"
                           .format(alignment))]
 
+        self.contains_reflections = {}
+        c_false = L.LiteralBool(False)
         for element, id in self.ir.element_ids.items():
-            parts += get_vector_reflection_array(L, self.ir.element_dof_reflection_entities[element],
-                                                 "ref_dof" + str(id))
-
+            vname = "ref_dof" + str(id)
```

Comment:
Make this part which generates a reflections array a member function of IntegralGenerator. The reason is to have the same granularity of code in generate() methods - if you have a look around this code there are only codes to other generate routines.